cristin-import/src/test/java/no/unit/nva/cristin/mapper/CristinMapperTest.java

```diff
@@ -280,6 +282,19 @@ void mapReturnsResourceWhereNvaContributorsNamesAreConcatenationsOfCristinFirstA
         assertThat(actualContributorNames, containsInAnyOrder(expectedContributorNames.toArray(String[]::new)));
     }
 
+    @Test
+    void shouldMapCristinContributorsWithoutSequenceNumberToNvaContributorsWithUniqueSequenceNumber() {
+        var singleCristinObject = cristinObjectWithSomeContributorsWithoutSequenceNumber(getSingleCristinObject());
+        List<Contributor> actualContributors = singleCristinObject.toPublication().getEntityDescription().getContributors();
+        List<Integer> actualSequenceNumbers = actualContributors.stream()
+                                    .map(Contributor::getSequence)
+                                    .collect(Collectors.toList());
+        var sequenceNumberSet = new HashSet<>(actualSequenceNumbers);
+        assertThat(actualContributors.size(), is(equalTo(sequenceNumberSet.size())));
+        assertThat(actualSequenceNumbers, is(equalTo(new ArrayList<>(sequenceNumberSet))));
+        assertThat(actualContributors, not(hasItem(nullValue())));
```

Comment:
Gjerne også assert at ikke to contributors har fått samme sekvens nummber

Reply:
Det gjøres faktisk en sjekk på det når vi asserter følgende, da sequenceNumberSet vil fjerne alle duplikater,
som også betyr at det ikke er dupliserte sequnceNumbers hos alle contributors:

###         
assertThat(actualContributors.size(), is(equalTo(sequenceNumberSet.size())));